spec/lib/job_notifier/notifier_spec.rb

```diff
@@ -109,7 +109,7 @@ def perform_with_feedback(_param1, _param2)
 
     context "with no owner" do
       before do
-        class ImageUploadJob < ActiveJob::Base
+        class ImageUploadJob < ApplicationJob
```

Comment:
⚠️ **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Lint/ConstantDefinitionInBlock: Do not define constants this way within a block.